test/ui/SearchAlertsTest.ts

```diff
@@ -308,10 +308,14 @@ export function SearchAlertsTest() {
           test.cmp.openPanel().then(() => {
             const elementOpened = getRootElementOpenedInModalBox(test);
             const dropdown = $$(elementOpened).find('.coveo-dropdown') as HTMLSelectElement;
+            const option = $$(dropdown).find('option[value="sunday"') as HTMLOptionElement;
+
+            option.selected = true;
+
             $$(dropdown).trigger('change');
             expect(updateSubscription).toHaveBeenCalledWith(
               jasmine.objectContaining({
-                frequency: 'monday',
+                frequency: 'sunday',
```

Comment:
So i'm guessing the UT was passing because we we were *always* passing "monday" (the first value in the dropdown) :man_facepalming:

Reply:
Yes exactly :sob: